front/app/components/ConfigurationMap/components/MapHelperOptions.tsx

```diff
@@ -92,7 +92,7 @@ const MapHelperOptions = ({ mapConfig, mapView }: Props) => {
           content={formatMessage(messages.setAsDefaultMapView)}
           hideOnClick={true}
         >
-          <div>
+          <div id="e2e-save-current-extent">
             <Button
```

Comment:
For some reason, data-cy doesn't always work in my e2e tests :thinking: I've used data-cy where I can, but some of these needed to be an "id". 

I've added a note for our next FE meeting to discuss if anyone else has the same issue.

Reply:
I think it's mostly been divs or components which aren't Buttons :thinking: